build.gradle

```diff
@@ -10,6 +10,8 @@ plugins {
   id 'uk.gov.hmcts.java' version '0.12.2'
 }
 
+ext['spring-security.version'] = '5.3.4.RELEASE'
+
```

Comment:
Not used anywhere? (Or if it is required, can we add to the `versions` object further down?)